<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bread</class>
 <widget class="QDialog" name="bread">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выпечка</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Хлеб белый</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Батон высший сорт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Лепешка пшеничная</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Хлеб ржаной</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Сушки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Булочка с шоколадом</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Пирог с творогом</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
